master/buildbot/statistics/storage_backends/influxdb_client.py

```diff
@@ -58,17 +47,20 @@ def __init__(self, url, port, user, password, db, captures,
                                      self.password, self.db)
         self.inited = True
```

Comment:
Sorry about this but no, inited is used in the tests (which would make it public).